community_server/src/Controller/AppRequestsController.php

```diff
@@ -301,12 +303,16 @@ public function getBalance($session_id = 0)
         }
         
         $body['decay_date'] = $now;
+        $this->addAdminError("AppRequests", "getBalance", $body, $user['id']);
         $this->set('body', $body);
     }
     
     public function listTransactions($page = 1, $count = 25, $orderDirection = 'ASC', $session_id = 0)
     {
+        
         $this->viewBuilder()->setLayout('ajax');
+        $this->response = $this->response->withType('application/json');
+
```

Comment:
What have these changes to do with the setup of the apollo server. Do we need them? If so, then why?